hugo/layouts/index.html

```diff
@@ -0,0 +1,129 @@
+{{ partial "head.html" . }}
+{{ partial "header.html" . }}
+<section class="banner">
+	<div class="container">
+		<div class="row">
+			<div class="col-lg-8 col-md-8 col-sm-12">
+				<div class="welcome">
+					<h3>Hej! Hallå!</h3>
+					<h3>Hallo! Hei! Halló!</h3>
+					<br />
+					<h1>Welcome to</h1>
+					<h1>Cloud Native Nordics</h1>
+				</div>
+			</div>
+			<div class="col-lg-4 col-md-4 col-sm-12">
+				<img class="logo float-right center-block" src="snowflake_white_border.png" />
+			</div>
+		</div>
+	</div>
+</div>
+</section>
+
+<section class="bg-purple py-4">
+	<div class="container">
+		<div class="row">
+			<div class="col-sm-12 col-lg-6 align-self-center">
+				<h3>Join our Slack community</h3>
+			</div>
+			<div class="col-sm-12 col-lg-6">
+				<form class="mb-0">
+					<input type="email" class="form-control rounded form-control-lg" placeholder="Enter email..." />
```

Comment:
Did you find a solution to make the request to the slack-invite service? I might have missed it in all the files in this PR and given its 5:30 in the morning 🙈

Reply:
No I have not tested this yet, but it should be possible just to put the URL in as a get action and then the slack-invite can use the email in the get params.
So we just need to move the DNS and then we can set this up proper.